SingularityUI/app/styles/stylus/slaveUsage.styl

```diff
@@ -13,35 +13,46 @@
 	margin-left : -85px
 
 // SlaveAggregates
-.total-slaves, .total-tasks
-	padding-top : 20px
-
-.total-slaves #value, .total-tasks #value
-	font-size : 60px
-	text-align : center
+.aggregate
+	padding-top : 20px;
+	text-align : center;
 	color: #7C98B6;
 
-.total-slaves #label, .total-tasks #label, .avg-cpu #label, .avg-memory #label
-	font-size : 20px
-	text-align : center
-	font-weight: 300;
-	color: #7C98B6;
+	&.vcenter
+		position: relative;
+		transform: translateY(50%);
 
-.avg-cpu
-	margin-left : 10px
-	width : 160px
+	a:hover
+		text-decoration: none;
+		border-bottom: 1px dotted;
+
+	.value
+		font-size : 50px
 
-.avg-memory
+	.label, .graph .label
+		font-size : 20px
+		font-weight: 300;
+		color: #7C98B6;
+		max-width: 100%;
+		white-space: normal;
+
+.graph
 	margin-left : 10px
 	width : 160px
 
+small
```

Comment:
This would override the color of `<small>` tags everywhere. If you actually want to do that, move this rule out of the `slaveUsage.styl`, otherwise scope this override to just this page.